server/rest/src/main/java/org/infinispan/rest/resources/ServerResource.java

```diff
@@ -352,6 +364,61 @@ private CompletionStage<RestResponse> report(RestRequest restRequest) {
       );
    }
 
+   private CompletionStage<RestResponse> nodeReport(RestRequest restRequest) {
+      EmbeddedCacheManager cacheManager = invocationHelper.getProtocolServer().getCacheManager();
+      List<Address> members = cacheManager.getMembers();
+      if (members == null) {
+         return report(restRequest);
```

Comment:
Shouldn't we return a `NOT_FOUND` response in this state, as it means that the specified `nodeName` does not exist?

If the specified `nodeName` is the local node, then we can just return `report(restRequest)`.